client/src/featureform/serving.py

```diff
@@ -83,25 +85,44 @@ def training_set(self, training_set_name, training_set_variant):
 
         return self.convert_ts_df_to_dataset(label, trainingset_df)
 
+    def get_input_df(self, source_name, source_variant):
+        if self.db.is_transformation(source_name, source_variant) == "PRIMARY":
+            source = self.db.get_source_variant(source_name, source_variant)
+            df = pd.read_csv(str(source['definition']))
+        else:
+            df = self.process_transformation(source_name, source_variant)
+        return df
+
+    def sql_transformation(self, query):
+            inputs = re.findall("(?={{).*?(?<=}})", query)
```

Comment:
Make this a function, something like parse_query_inputs() so its more clear what this does